RELEASE-NOTES.md

```diff
@@ -17,10 +16,17 @@
 + The experimental JAX sampling was removed. It will be included in the upcoming `v4` release ([see #4866](https://github.com/pymc-devs/pymc3/pull/4866)).
 + `pm.intX` no longer downcasts integers unnecessarily (see [#4569](https://github.com/pymc-devs/pymc3/pull/4569))
 
+### Maintenance
++ A deprecation warning from the `semver` package we use for checking backend compatibility was dealt with (see [#4547](https://github.com/pymc-devs/pymc3/pull/4547)).
++ `theano.printing.pydotprint` is now hotfixed upon import (see [#4594](https://github.com/pymc-devs/pymc3/pull/4594)).
++ Fix bug in the computation of the log pseudolikelihood values (SMC-ABC). (see [#4672](https://github.com/pymc-devs/pymc3/pull/4672)).
++ Fix `LKJCorr.random` method to work with `pm.sample_prior_predictive`. (see [#4780](https://github.com/pymc-devs/pymc3/pull/4780)).
++ Enable documentation generation via ReadTheDocs for upcoming v3 releases. (see [#4805](https://github.com/pymc-devs/pymc3/pull/4805)).
++ Remove `float128` dtype support (see [#4834](https://github.com/pymc-devs/pymc3/pull/4834)).
++ Use `to_tuple` function in `pm.fast_sample_posterior_predictive` to pass shape assertions (see [#4927](https://github.com/pymc-devs/pymc3/pull/4927)).
 
 **Release manager** for 3.11.3: Ravin Kumar ([@canyon289](https://github.com/canyon289))
```

Comment:
i don't know how releases work, just curious if this line should be there if we're doing 3.11.4 and skipping 3.11..3

Reply:
ITs a good question. We technically did release 3.11.3 so if someone installs from github this is the correct changelog (now that I fixed the tags)

Since we have some docfixes to do below I think thatll end up being the extent of 3.11.4s changes

Reply:
I agree, I think we should just mark 3.11.3 as broken and move release notes up.

Reply:
Made the changes